monkestation/code/modules/cybernetics/augments/arm_augments/item_sets/jobs.dm

```diff
@@ -112,14 +112,13 @@
 		if(located)
 			located.forceMove(drop_location())
 
-/obj/item/organ/internal/cyberimp/arm/item_set/chemical
-	name = "chemical toolset implant"
-	desc = "A set of chemical tools hidden behind a concealed panel on the user's arm."
+/obj/item/organ/internal/cyberimp/arm/item_set/paramedic
+	name = "paramedic toolset implant"
+	desc = "A set of rescue tools hidden behind a concealed panel on the user's arm."
 	items_to_create = list(
-		/obj/item/reagent_containers/cup/beaker,
-		/obj/item/reagent_containers/cup/beaker,
-		/obj/item/reagent_containers/cup/beaker,
-		/obj/item/reagent_containers/dropper
+		/obj/item/roller/robo,
+		/obj/item/sensor_device,
+		/obj/item/gun/medbeam
 	)
 	encode_info = AUGMENT_NT_LOWLEVEL
```

Comment:
Personally i'd say a medibeam would probably at least be a highlevel encode? Being compatable with a 1.0 nt cyberlink (which could be roundstart for some people) w/o hacking seems off for me

Reply:
Fair point